packages/loot-design/package.json

```diff
@@ -24,7 +24,7 @@
     "node-noop": "1.0.0",
     "npm-run-all": "^4.1.3",
     "polished": "^1.8.1",
-    "prettier": "^1.14.2",
+    "prettier": "^2.8.2",
```

Comment:
Is this dependency needed?

Reply:
Nope. Removed this dependency from the individual packages. Now we have it only on the root level.
https://github.com/actualbudget/actual/pull/442/commits/afdfd3c6b188e160242ba5f8839d177ece53e7f1